devicemodel/core/main.c

```diff
@@ -1006,12 +1000,15 @@ main(int argc, char *argv[])
 	argc -= optind;
 	argv += optind;
 
-	if (argc != 1)
+	if (argc != 1) {
+		pr_err("You must provide the name of the Virtual Machine (VM) you want to start. Exiting.\n");
```

Comment:
```suggestion
		pr_err("You must provide the name of the Virtual Machine (VM) you want to start. Exiting.");
```
?

Reply:
`pr_err` for dm is different from hv, `\n` is neccessary.

Reply:
Oh, OK